R/iterate_hector.R

```diff
@@ -105,7 +105,9 @@ metric_calc_1run <- function(x, metric) {
 #' h_result <- iterate_hector(core, params)
 #' head(h_result)
 
-iterate_hector <- function(core, params) {
+iterate_hector <- function(core, params,
+                           save_years = 1745:2300,
```

Comment:
Better might be to query the core for its years...? E.g. have NULL here and then later on do
```r
if(is.null(save_years)) {
  save_years <- core$strtdate : core$enddate
}
```

Reply:
That makes sense, otherwise I see how there would be issues hard coding the default year range if the user's core doesn't have a date range matching the default.